linux_os/guide/services/base/service_cgconfig_disabled/rule.yml

```diff
@@ -14,14 +14,15 @@ rationale: |-
     Unless control groups are used to manage system resources, running the cgconfig
     service is not necessary.
 
-severity: unknown
+severity: low
 
 identifiers:
     cce@rhel6: 27250-0
     cce@rhel7: 80254-6
+    cce@rhel8: 82393-0
```

Comment:
@shawndwells we should return this CCE back to the pool. RHEL8 doesn't appear to have `libcgroup` package anymore. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index

Reply:
Done. Also removed ``rhel8`` from prodtype for this rule.